sql: correct CLUSTER SETTING hint for sql.defaults.datestyle.enabled

otan opened the pull request:
Release note (bug fix): Fix an incorrect bug hint for
`sql.defaults.datestyle.enabled`.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/70842)
<!-- Reviewable:end -->

otan committed:
sql: correct CLUSTER SETTING hint for sql.defaults.datestyle.enabled

Release note (bug fix): Fix an incorrect bug hint for
`sql.defaults.datestyle.enabled`.

otan commented:
bors r=rafiss
